src/main/resources/db/migration/V1.0.0__initial_schema.sql

```diff
@@ -0,0 +1,451 @@
+-- TODO: Think about image storing and referencing
+-- TODO: Moderator feedback
+-- TODO?: Email tasks
+
+create type actor_role as enum ('REVIEWER', 'HR', 'MODERATOR', 'ADMINISTRATOR', 'BRAND_PRESENTER');
+
+create table image(
+    id uuid primary key,
+    location varchar not null
+);
+
+create table actor
+(
+    id                 uuid primary key,
+    first_name         varchar(64) not null,
+    last_name          varchar(64) not null,
+    email              varchar(320) unique, -- TODO: Email verification on registration, on update
+    role               actor_role not null,
+    password           varchar(60) not null,
+    created_timestamp  timestamp not null,
+    profile_image_id   uuid references image,
+    is_active          boolean not null
+);
+
+create table email_confirmation_token
+(
+    token             varchar(73) primary key,
+    actor_id          uuid not null references actor,
+    email             varchar(320) not null unique,
+    created_timestamp timestamp not null
+);
+create index email_confirmation_token_created_timestamp_idx ON email_confirmation_token (created_timestamp);
+
+create table password_recovery_token
+(
+    token             varchar(73) primary key,
+    actor_id          uuid not null references actor,
+    created_timestamp timestamp not null
+);
+create index password_recovery_token_created_timestamp_idx ON password_recovery_token (created_timestamp);
+
+
+create table brand
+(
+    id          uuid primary key,
+    name        varchar(64) not null,
+    description text,
```

Comment:
Let's use either text or varchar without explicit length limitation in such cases.

Reply:
Ok, let's use varchar